webapp-django/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env node */
+const path = require('path');
+const BundleTracker = require('webpack-bundle-tracker');
+
+module.exports = {
```

Comment:
This file is optimized for development. If we move forward with the reimplmented page, we should revisit this file and make sure the bundles it's building are small and fast.

Reply:
For things like this since we collectively forget unless there's a string tied around someone's finger, we should either create a Bugzilla bug after this lands or add a FIXME in the comments here or both.